src/features/groups/lib/actions.ts

```diff
@@ -64,3 +67,27 @@ export const removeMember = async (
   if (result.ok) return Ok(undefined);
   return Err(result.error);
 };
+
+/**
+ * Function to validate input, if valid, call the API client to create a new container
+ * @param containerNameInput {@link CreateContainerInputs}The user input to be validated
+ * @param groupId An identifier of a group which a new container will belong to.
+ * @returns Undefined for success, an error message if validation or request fails
+ */
```

Comment:
When adding a `@param` tag, here is the right format:
```ts
/**
 * @param <param_name> - (hyphen) <description>
 * @param groupId - An identifier of a group which a new container will belong to.
 */
```

https://tsdoc.org/pages/tags/param/